yb-voyager/cmd/importData.go

```diff
@@ -508,6 +508,13 @@ func truncateTables(tables []string) {
 		if err != nil {
 			utils.ErrExit("error while truncating table %q: %s", table, err)
 		}
+
+		cmd := fmt.Sprintf(`DELETE FROM ybvoyager.batches WHERE file_name LIKE '%s\.%%'`, table)
```

Comment:
Do we need to escape `.`.

Reply:
Should we maintain table_name in the `batches` table?

Reply:
The escape character in the query makes no difference. Nevertheless removing it.
```
dvdrental=# select * from ybvoyager.batches where file_name like 'film%';
             file_name             | rows_imported
-----------------------------------+---------------
 film.0.1000.1000.333094.P         |          1000
 film_category.0.1000.1000.26321.P |          1000
 film_actor.0.5462.5462.149469.P   |          5462
(3 rows)

dvdrental=# select * from ybvoyager.batches where file_name like 'film\.%';
         file_name         | rows_imported
---------------------------+---------------
 film.0.1000.1000.333094.P |          1000
(1 row)

dvdrental=# select * from ybvoyager.batches where file_name like 'film.%';
         file_name         | rows_imported
---------------------------+---------------
 film.0.1000.1000.333094.P |          1000
(1 row)
```

About keeping table name in the `batches` table: Let's do it only when we know that scanning the batches table is a serious perf issue.